quconot/implementations/mct_recursion.py

```diff
@@ -14,6 +14,13 @@
 
 
 class MCTRecursion(MCTBase):
+    r"""
+    Implementation of the multi-controlled not gate with no auxiliary qubits using Qiskit’s ``recursion`` mode. This
+    is implemented using MCXRecursive. Using one auxiliary qubit, the MCT gate is recursively split onto four

```

Comment:
qiskit ones are not implementations. they are directly coming from qiskit

change wording for all qiskit ones